src/index.js

```diff
@@ -1,2 +1,26 @@
 import './style.scss';
 import './assets/logo.png';
+import { addDataToApi, generateLeaderBoard } from './modules/leaderboard.js';
+import Score from './modules/Score.js';
+
+const scoreList = document.querySelector('#scoreList');
+const addScoreForm = document.querySelector('#addScoreForm');
+const refreshScoresBtn = document.querySelector('#refreshScoresBtn');
+
+addScoreForm.addEventListener('submit', (e) => {
+  e.preventDefault();
+  const data = new FormData(e.target);
+  const newScore = new Score({
+    user: data.get('user'),
+    score: data.get('score'),
+  });
+  addDataToApi(newScore);
+  addScoreForm.reset();
+});
```

Comment:
- [ OPTIONAL] It would be nice to add some sort of check that ensures the value of the _**name**_ and _**score**_ is not empty or null before submitting to the `POST` endpoint